server/server.lua

```diff
@@ -31,8 +32,18 @@ lib.callback.register('npwd_qbx_garages:server:getPlayerVehicles', function(sour
 			vehicleData.brand = VEHICLES[model].brand
 		end
 
-		vehicleData.garage = config.garages[vehicleData.garage]?.label or locale('states.garage_unknown')
+		vehicleData.garage = garageConfig[vehicleData.garage]?.label or locale('states.garage_unknown')
 	end
 
 	return result
+end)
+
+AddEventHandler('onResourceStart', function(resourceName)
+    if resourceName == 'qbx_garages' then
+        garageConfig = exports.qbx_garages:GetGarages()
+    end
```

Comment:
This seems redundant as we are calling this same export at top of file

Reply:
This is to catch restarts of qbx_garages without kicking this resource